torch_geometric/transforms/radius_graph.py

```diff
@@ -17,8 +17,8 @@ class RadiusGraph(BaseTransform):
             with message passing (:obj:`"source_to_target"` or
             :obj:`"target_to_source"`). (default: :obj:`"source_to_target"`)
     """
-    def __init__(self, r, loop=False, max_num_neighbors=32,
-                 flow='source_to_target'):
+    def __init__(self, r: float, loop: bool = False,
```

Comment:
another coding style would be each argument take a new line so that in the future if we add a new argument, only the new argument line changes.

Reply:
Resolved.